ckanext/security/validators.py

```diff
@@ -58,7 +59,10 @@ def user_about_validator(key, data, errors, context):
 
 invalid_list = ['hacked', 'hack[^a-zA-Z]+by']
```

Comment:
@jguo144 We should 1) add 'hack' to the list 2)decapitalize the `input_value` to prevent users from using names like 'JunTheHacker' or "Hacking fun"